### ISSUE TITLE
Biome is not recognized as a formatter

### ISSUE THREAD

**Issue by jansedlon** [farreldarian/prisma-generator-drizzle#54]
Hey, when i want to use biome as a formatter, it throws this error

```log
@flixydev/prisma:db:generate: [prisma-generator-drizzle] Invalid Config:
@flixydev/prisma:db:generate: 
@flixydev/prisma:db:generate: - formatter: Invalid type: Expected "prettier" but received "biome"
```

**Comment by jansedlon** [comment]
I see, you have added it in main branch, but it's not released yet :) I'll make a patch for myself

**Comment by farreldarian** [comment]
Hi @jansedlon, many apologies for this, I think I was preparing for another feature but got distracted. Here I just released [v0.7.2](https://github.com/farreldarian/prisma-generator-drizzle/releases/tag/v0.7.2) to push this feature

**Comment by jansedlon** [comment]
That's great, thank you!
